Products/Zuul/facades/manufacturersfacade.py

```diff
@@ -73,6 +73,8 @@ def editProduct(self, params=None):
         prodName = params['oldname']
         prod = manufacturer.products._getOb(prodName, None)
         if prod:
+            instances = [i.device() for i in prod.instances()]
```

Comment:
if it possible to use here "generator comprehension"?
E.g. (i.device() for i in prod.instances())

Reply:
Unfortunately we need to store information about instances before product is ie. renamed. After renaming UID of product is changed and data about instances is lost.

Reply:
got you